resources/public/api/conf/1.0/sandbox/root-response.md

```diff
@@ -14,7 +14,7 @@
         <td><p>Benefits and Credits data found </p>
         <td>
             <p>matchId=&lt;obtained from Individuals Matching API example: 57072660-1df9-4aeb-b4ea-cd2d7f96e430&gt;</p>
-            <p>startTaxYear=2018-19<br>endTaxYear= 2019-20</p>
+            <p>fromTaxYear=2018-19<br>toTaxYear= 2019-20</p>
```

Comment:
I think the query parameters are fromDate and toDate - can you please check throughout, because I think there's some variation....

Reply:
See for example "startTaxYear" in sandbox/child-tax-credits-response.md